modules/motions/Straight.h

```diff
@@ -0,0 +1,49 @@
+/**
+ * @file   Straight.h
+ * @brief  直進動作の親クラス
+ * @author HaruArima08
+ */
+
+#ifndef STRAIGHT_H
+#define STRAIGHT_H
+
+#include "Motion.h"
+#include "Pid.h"
+#include "Mileage.h"
+#include "SpeedCalculator.h"
+
+class Straight : public Motion {
+ public:
+  /**
+   * @brief コンストラクタ
+   * @param _robot ロボット本体への参照
+   * @param _speed   目標速度[mm/s]
```

Comment:
細かいけど、speedの方に空白が多いのが気になった
```suggestion
   * @param _robot ロボット本体への参照
   * @param _speed 目標速度[mm/s]
```